terraform/gcp/outputs.tf

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
-output "service_account_email" {
-  value = google_service_account.autoneg.email
+output "service_account" {
```

Comment:
Can you keep `service_account_email` to keep backwards compatibility?

Reply:
Sure thing, I've added it back @rosmo